services/checks/checks.go

```diff
@@ -451,8 +452,36 @@ func (s *Service) minPMMAgentVersion(t check.Type) *version.Parsed {
 	}
 }
 
-// executeChecks runs all available checks for all reachable services.
-func (s *Service) executeChecks(ctx context.Context) error {
+func (s *Service) filterChecks(checks []check.Check, disable, enable []string) []check.Check {
+	var res []check.Check
+	dm := make(map[string]struct{}, len(disable))
+	for _, e := range disable {
+		dm[e] = struct{}{}
+	}
+
+	em := make(map[string]struct{}, len(enable))
+	for _, e := range enable {
+		em[e] = struct{}{}
+	}
+
+	for _, c := range checks {
+		if _, ok := dm[c.Name]; ok {
+			s.l.Debugf("Skipping disabled mySQL check %s", c.Name)
```

Comment:
it's generic check, or? why is there mysql

Reply:
It was copy-pasted. Fixed 👍